### ISSUE TITLE
RK3588一个NPU的核可以同时跑多个yolov5模型么？

### ISSUE THREAD

**Issue by sunian6183** [airockchip/rknn-toolkit2#155]
RK3588的一个NPU的核在跑一个yolov5模型的时候利用率并没有达到100%，只有30%左右。可以同时跑多个模型使利用率达到100%么？

**Comment by yuyun2000** [comment]
你可以试试，应该是可以的

**Comment by swdee** [comment]
Yes, you can run multiple of the same model.  We do [this here](https://github.com/swdee/go-rknnlite/tree/master/example/pool).
